src/index.html

```diff
@@ -7,6 +7,38 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Calendar</h1>
+    <div class="calendar calendar--month-length-31">
+      <div class="calendar__day-box calendar__day-box--day-1 calendar__day-box--wed"></div>
+      <div class="calendar__day-box calendar__day-box--day-2"></div>
```

Comment:
```suggestion
    <div class="calendar calendar--month-length-31 calendar--start-day-mon">
      <div class="calendar__day"></div>
      <div class="calendar__day"></div>
```

Reply:
classes should be like this, avoid redundant classes, and re-write styles according to this classes